clients/drcachesim/tests/invariant_checker_on_regdeps_trace.templatex

```diff
@@ -0,0 +1,10 @@
+Hello, world!
```

Comment:
> the checks for 0 read/write records at the end of trace processing, as we can't adjust the counters accordingly because we can't infer the precise number of load/stores performed by DR_ISA_REGDEPS instructions;

This is not really a check: this is just part of the correct # read/write checks.  Move those to be part of that read/write bullet below.

Reply:
Done.